src/ServicePulse.Host/app/js/views/archive/controller.js

```diff
@@ -72,15 +73,29 @@
             vm.loadingData = false;
         };
 
+        var storeGroupOption = function (amount, unit) {
+            $cookies.put('archive_amount', amount);
+            $cookies.put('archive_unit', unit);
+        }
+
+        var loadDefaultGroup = function() {
+            var amount = $cookies.get("archive_amount");
+            var unit = $cookies.get("archive_unit");
+
+            vm.timeGroup.amount = amount;
+            vm.timeGroup.unit = unit;
+        }
+
         var init = function () {
 
             vm.configuration = sharedDataService.getConfiguration();
             vm.error_retention_period = moment.duration(vm.configuration.data_retention.error_retention_period).asHours();
             vm.total = 1;
             vm.archives = [];
             vm.sort.page = 1;
-            vm.sort.start = moment.utc().subtract(vm.timeGroup.amount, vm.timeGroup.unit).format('YYYY-MM-DDTHH:mm:ss');
-            vm.sort.end = moment.utc().format('YYYY-MM-DDTHH:mm:ss');
+
+            loadDefaultGroup();
+            vm.selectTimeGroup(vm.timeGroup.amount, vm.timeGroup.unit);
```

Comment:
This method also sets the `vm.timeGroup.amount` and `vm.timeGroup.unit` variables. Is it worth refactoring this so that the load method doesn't set the variables but rather returns it's own object that gets passed in as a variable here?